puddlestuff/masstag/dialogs.py

```diff
@@ -369,10 +369,10 @@ def remove(self):
     def setProfile(self, profile):
         self._tsps = [tsp for tsp in profile.profiles if tsp.tag_source]
         [self.listbox.addItem(tsp.tag_source.name) for tsp in self._tsps]
-        self.albumBound.setValue(profile.album_bound * 100)
+        self.albumBound.setValue(int(profile.album_bound * 100))
```

Comment:
should we use `round()` instead of `int()` to avoid cases like:

```
In [3]: n = 99.9
In [4]: int(n)
Out[4]: 99
In [5]: round(n)
Out[5]: 100
```

alternatively we could also store the percentage as an int 0..100 instead of a float